website/addons/dropbox/utils.py

```diff
@@ -43,10 +43,14 @@ def log(self, action, extra=None, save=False):
         :param dict extra: Extra parameters to add to the ``params`` dict of the
             new NodeLog.
         """
+        if self.node.get_addon('dropbox', deleted=True).folder == '/':
+            folder = '/ Full DropBox'
```

Comment:
To clarify: I don't think this "display name" should be stored in the logs. The display logic should happen in the front-end.